### PR TITLE
Sort reviews on user show

### PR DESCRIPTION
**Pull request by bplantico**
What does this PR do?

This pull request adds the ability to sort reviews by date created (newest or oldest) on a user's show page.

### CODE REVIEW

File: spec/features/users/show_spec.rb [hunk 1]
```diff
@@ -138,6 +138,31 @@
...
 
+    it "there are links to sort reviews by newest first and oldest first" do
+      user_1 = User.create!(name: "User One")
+
+      review_1 = @book_1.reviews.create!(title: "First", rating: 1, body: "stuff about book 1", user: user_1)
+      review_2 = @book_2.reviews.create!(title: "Second", rating: 2, body: "stuff about book 2", user: user_1, created_at: "2019-05-12 02:53:27.326667")
+
+      visit user_path(user_1.id)
+
+      expect(page).to have_link("Sort Reviews (newest first")
```
Comment (hsmitha26) [inline spec/features/users/show_spec.rb]: Is this link working?

Shouldn't this be: expect(page).to have_link("Sort Reviews (newest first)")?

**Review by hsmitha26 — COMMENTED** [review]
Please see lines: 149, 150, 161

Shouldn't this be: expect(page).to have_link("Sort Reviews (newest first)")?

File: spec/features/users/show_spec.rb (hunk 1 above)
Comment (bplantico) [reply spec/features/users/show_spec.rb]: Went back and corrected those three lines, tests are all passing! :)

**Review by hsmitha26 — APPROVED** [review]
Looks great, thank you for all your hard work!

**Pull request merged**